### ISSUE TITLE
exclude rds and elasticache from ec2.ini

### ISSUE THREAD

**Issue by gavD** [radify/radiian#58]
ec2.ini should include:

```ini
# To exclude RDS instances from the inventory, uncomment and set to False.
rds = False

# To exclude ElastiCache instances from the inventory, uncomment and set to False.
elasticache = False
```

Right now, it looks like:

```ini
# To exclude RDS instances from the inventory, uncomment and set to False.
#rds = False

# To exclude ElastiCache instances from the inventory, uncomment and set to False.
#elasticache = False
```

**Comment by jfmercer** [comment]
@gavD Should I ask the user to set these values during the QA?

**Comment by gavD** [comment]
no. Elasticache and RDS have nothing to do with radiian. It's not a big
deal but we might as well disable them from the dynamic inventory

On Tue, Dec 8, 2015 at 5:21 PM, John Mercer <notifications@github.com>
wrote:

> @gavD <https://github.com/gavD> Should I ask the user to set these values
> during the QA?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/radify/radiian/issues/58#issuecomment-162951398>.
>



-- 
Gavin Davies | Automation Engineer; Radify, Inc. | radify.io

**Comment by jfmercer** [comment]
Closed by #64
